src/tools/fuzzing/fuzzing.cpp

```diff
@@ -2450,11 +2448,21 @@ Expression* TranslateToFuzzReader::makeRefFuncConst(Type type) {
     // had generic 'func' here.
     heapType = Signature(Type::none, Type::none);
   }
-  // TODO: randomize the order
-  for (auto& func : wasm.functions) {
-    if (Type::isSubType(Type(func->type, NonNullable), type)) {
-      return builder.makeRefFunc(func->name, func->type);
-    }
+  // Look for a proper function starting from a random location, and loop from
+  // there, wrapping around to 0.
+  if (!wasm.functions.empty()) {
+    Index start = upTo(wasm.functions.size());
+    Index i = start;
+    do {
+      auto& func = wasm.functions[i];
+      if (Type::isSubType(Type(func->type, NonNullable), type)) {
+        return builder.makeRefFunc(func->name, func->type);
+      }
+      i++;
+      if (i == wasm.functions.size()) {
+        i = 0;
+      }
```

Comment:
```suggestion
      ++i %= wasm.functions.size();
```
or perhaps more conventionally
```suggestion
      i = (i + 1) % wasm.functions.size();
```

Reply:
> `++i%=`

Looks like perl!